src/autoscaler/scalingengine/scalingengine.go

```diff
@@ -330,6 +331,11 @@ func (s *scalingEngine) RemoveActiveSchedule(appId string, scheduleId string) er
 
 	appEntity, err := s.cfClient.GetApp(appId)
 	if err != nil {
+		if _, ok := err.(*models.AppNotFoundErr); ok {
+			logger.Error("app-not-found", err)
+			history.Status = models.ScalingStatusIgnored
+			return nil
```

Comment:
Why still log error here ... maybe just log info here since it is a type of expected error.